src/test/java/chips/ChipsClientTest.java

```diff
@@ -0,0 +1,33 @@
+package chips;
```

Comment:
Package is wrong here. Needs to match where the class is and the ChipsClient import statement below should then not be needed.

Reply:
Refactored